### PR TITLE
[ADD][11.0] connector_infor_account_move module

### PR DESCRIPTION
**Pull request by bodedra**
Review description https://github.com/OCA/connector-infor/issues/2

Depends on:

- [ ] https://github.com/OCA/connector-infor/pull/3

### CODE REVIEW

**Comment by bodedra** [comment]
CC @max3903 @guewen

File: connector_infor_account_move/models/backend_infor_journal_custom_field.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class BackendInforJournalCustomField(models.Model):
+    _name = 'backend.infor.journal.custom.field'
+    _description = 'Backend Infor Journal Custom Field'
+
+    Name = fields.Char(string='Name')
```
Comment (guewen) [inline connector_infor_account_move/models/backend_infor_journal_custom_field.py]: s/Name/name/

File: connector_infor_account_move/models/backend_infor_journal_custom_field.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    type = fields.Selection(
```
Comment (guewen) [inline connector_infor_account_move/models/backend_infor_journal_custom_field.py]: `type` is a python builtin, better use `field_type` or `kind`, or anything else

File: connector_infor_account_move/models/backend_infor_journal_line.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class BackendInforJournalLine(models.Model):
+    _name = 'backend.infor.journal.line'
```
Comment (guewen) [inline connector_infor_account_move/models/backend_infor_journal_line.py:8]: why `line`?

**Review by guewen — COMMENTED** [review]
Small details

File: connector_infor_account_move/models/account_invoice.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+    _inherit = 'account.invoice'
+
+
+    @api.multi
+    def invoice_validate(self):
+        template = Template(body_template)
+        today = fields.Datetime.now(pytz.timezone('GMT'))
+        for invoice in self:
+            # Only applicable for Vendor bills
+            if invoice.type == 'in_invoice':
```
Comment (max3903) [inline connector_infor_account_move/models/account_invoice.py:102]: @bodedra @guewen it's applicable for any posted journal entry in the journal configured on the backend.

**Comment by guewen** [comment]
Replaced by #5

**Pull request closed without merging**